flight_computer.ino

```diff
@@ -9,6 +9,7 @@
 */
 
 #include <Wire.h>
+#include <SD.h>
```

Comment:
Do we need this?

Reply:
This is not included in the master branch, looks like you have not pulled master and resolved conflicts.

Reply:
Alright, I've pulled again but it didn't change, so I've manually removed it and pushed the changes.